<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoveListWidget</class>
 <widget class="QWidget" name="MoveListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Moves</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_GoBack1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Previous move</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="icon">
        <iconset theme=":/icons/toolbutton/previous_16x16"/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_GoBack2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Back 1 full move (2 plies)</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="icon">
        <iconset theme=":/icons/toolbutton/previous_16x16"/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_GoStart">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Skip to the beginning</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme=":/icons/toolbutton/first_16x16"/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btn_GoNext">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Next move</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme=":/icons/toolbutton/next_16x16"/>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="btn_GoCurrent">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Current position</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme=":/icons/toolbutton/last_16x16"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QScrollArea" name="scrollArea">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="widget_Moves">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>302</width>
            <height>300</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QTextBrowser" name="m_moveList">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="openLinks">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_Moves">
     <item>
      <widget class="QPushButton" name="btn_CopyFen">
       <property name="text">
        <string>Copy FEN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_CopyPgn">
       <property name="text">
        <string>Copy PGN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_CopyZS">
       <property name="toolTip">
        <string>Copy Zobrist signature</string>
       </property>
       <property name="text">
        <string>Copy ZS</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
